src/reducers/url-state.js

```diff
@@ -17,6 +17,14 @@ import type {
 import type { UrlState, Reducer } from '../types/state';
 import type { TabSlug } from '../app-logic/tabs-handling';
 
+/*
+ * Note: all reducers in this state can be shortcut and the state reset
```

Comment:
I think "shortcut" should be "cut short".

Reply:
From what I see, there are "short-cut" and "cut through", I'll use the latter.

Reply:
So, I used `circumvented` :)